components/button/button.js

```diff
@@ -12,19 +12,18 @@ function Button(props) {
   const {
     children,
     className,
-    size,
+    dataAttrs = {},
+    disabled,
     href,
+    id,
     onClick,
+    size,
     type,
     variation,
-    disabled,
-    dataAttrs = {},
   } = props;
   const restProps = getDataAttributes(dataAttrs);
   const mods = props.mods ? props.mods.slice() : [];
 
-  const otherProps = ejectOtherProps(props, Button.propTypes);
```

Comment:
Why are you removing `otherProps`?

Reply:
It is a decision to not have `otherProps` anymore passed in ui-kit.. This opens vulnerability to pass properties that are not suitable for the DOM and also does allow a lot of freedom. The decision is, in future to pass only known properties - that are publicly documented in the ui-kit. If there is a need to extend the current list of properties - please make sure that proper PRs are added.